### ISSUE TITLE
Initial label embeddings

### ISSUE THREAD

**Issue by Priptide** [seanhe97/MKT#10]
Hey, I was wondering how you generated the label embeddings which are in the other files section used for the first training round as I couldn't see in the paper how these were generated unless it is just using the clip ViT model?

**Comment by sunanhe** [comment]
The label embeddings in the first training stage are generated by the bert of  pretrained CLIP model.

Best,
Sunan
